pkgs/applications/editors/kakoune/plugins/generated.nix

```diff
@@ -41,16 +41,41 @@ let
 
   fzf-kak = buildKakounePluginFrom2Nix {
     pname = "fzf-kak";
-    version = "2021-03-15";
+    version = "2021-03-19";
     src = fetchFromGitHub {
       owner = "andreyorst";
       repo = "fzf.kak";
-      rev = "4e6c9a857511fccdbbc835a1c9acb205b6486a4c";
-      sha256 = "0syhhdlsm7vg6hcd2n2acag9g562z49rbb5smh5p2gnplhmp93i0";
+      rev = "435cc240f3732770b19cc7dce9bdf65e22d6ed6a";
+      sha256 = "0ahcz7z5i8bngjnkd6yr782wah873yxgm8xrwa9g8gkddvj3q6rw";
     };
     meta.homepage = "https://github.com/andreyorst/fzf.kak/";
   };
 
+  kak-tree = buildKakounePluginFrom2Nix {
```

Comment:
This plugin needs to be compiled with rust, so it should be either on the blocklist or added with buildRustPackage.

Reply:
Thank you, I also realized this as well. I haven't had as much time as I'd like, but I should have a few hours tomorrow to incorporate some of your suggestions and clean this up.